apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2

```diff
@@ -271,6 +271,36 @@
                 }
             },
             {% endif %}
+            {% if console.internal_notification_sending is defined %}
+            "internalNotificationSending": {
+                "disabledFeatures": [
+                    {% if console.internal_notification_sending.disabled_features is defined %}
+                    {% for feature in console.internal_notification_sending.disabled_features %}
+                    "{{ feature }}"{{ "," if not loop.last }}
+                    {% endfor %}
+                    {% endif %}
+                ],
+                "enabled": {% if console.internal_notification_sending.enabled is defined %} {{ console.internal_notification_sending.enabled }},
+                {% else %} true,
+                {% endif %}
+                "scopes": {
+                    {% if console.internal_notification_sending.scopes is defined %}
+                    {% for operation, scopes in console.internal_notification_sending.scopes.items() %}
+                    "{{ operation }}": [
+                        {% for scope in scopes %}
+                            "{{ scope }}"{{ "," if not loop.last }}
+                        {% endfor %}
+                    ]{{ "," if not loop.last }}
+                    {% endfor %}
+                    {% else %}
+                        "create": [],
+                        "read": [],
+                        "update": [],
```

Comment:
Shall we add the "feature" scope as well?

Reply:
Fixed b4d854d26bb3c124f47304139dd7ebff4bb856c6